sdk/core/ci.yml

```diff
@@ -89,6 +89,9 @@ stages:
         - Name: azure-core-tracing-opentelemetry
           Path: azure-core-tracing-opentelemetry
           VcpkgPortName: azure-core-tracing-opentelemetry-cpp
+        - Name: azure-core-amqp
+          Path: azure-core-amqp
+          VcpkgPortName: azure-core-amqp-cpp
```

Comment:
Not a bad idea, but today, we currently have a single code coverage target for each service (because our CI pipelines are instanced on a per-service basis).

I believe that adding per-package CI pipelines would require redesigning how our nightly builds work to include per-package pipelines. 

Completely possible but a lot of work. And I'm not 100% sure if there's any significant benefit to doing that. It's also not how any of the other languages do this (every other language has per-service CI pipelines, not per-package CI pipelines).